install.sh

```diff
@@ -71,6 +71,45 @@ if (($SUPPORTS_SSE42 == 0)); then
     exit 1
 fi
 
+replace_tsdb() {
+    if (
+        [ -f "$SENTRY_CONFIG_PY" ] &&
```

Comment:
You should not need to check this as we ensure the file exists before proceeding?